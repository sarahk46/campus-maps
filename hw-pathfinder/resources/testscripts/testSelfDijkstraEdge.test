# Find a path with Dijkstra's algorithm between
# one node to itself where there is an edge.
# Tests that this holds even when there is a weighted edge.

CreateGraph simpleGraph

AddNode simpleGraph A

AddEdge simpleGraph A A 10.0

FindPath simpleGraph A A