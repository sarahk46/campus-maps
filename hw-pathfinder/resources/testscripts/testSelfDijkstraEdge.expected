# Find a path with Dijkstra's algorithm between
# one node to itself where there is an edge.
# Tests that this holds even when there is a weighted edge.

created graph simpleGraph

added node A to simpleGraph

added edge 10.000 from A to A in simpleGraph

path from A to A:
total cost: 0.000